lua/controller.lua

```diff
@@ -332,9 +334,11 @@ local function controller_allow_metadata_inventory_put(pos, listname, index, sta
 
 	if drawer_net_index[stack:get_name()] then
 		local drawer = drawer_net_index[stack:get_name()]
+	local meta = core.get_meta(pos)
```

Comment:
Bad indent level

Reply:
I agree and I only used it here to match the existing code. I run a search/replace to migrate all instances

Reply:
Done

Reply:
I personally prefer `core.`. I'm using it in my mods because it's independent of the game name and shorter. Please don't replace this in the whole mod.

Reply:
So minetest.* require minetest game and core does not? Is there any functional difference?

Reply:
No, it's the same table, just not advertised anywhere. I did not notice that `core` was already used in the entire mod, but since that's the case, it's better to continue like that.

Reply:
understood

Reply:
I put them all back to core.